rmw_zenoh_cpp/src/detail/attachment_helpers.hpp

```diff
@@ -17,19 +17,27 @@
 
 #include <zenoh.h>
 
-#include <string>
-
 #include "rmw/types.h"
 
 namespace rmw_zenoh_cpp
 {
-//==============================================================================
-bool get_gid_from_attachment(
-  const z_attachment_t * const attachment, uint8_t gid[RMW_GID_STORAGE_SIZE]);
 
-//==============================================================================
-int64_t get_int64_from_attachment(
-  const z_attachment_t * const attachment, const std::string & name);
+class attachement_data_t final
```

Comment:
```suggestion
class attachment_data_t final
```

Reply:
Fixed in https://github.com/ros2/rmw_zenoh/pull/276/commits/9f78ca46344677a0a36d9c6fd651ee8e620f51b0